integration-cli/docker_cli_volume_test.go

```diff
@@ -299,3 +299,15 @@ func (s *DockerSuite) TestVolumeCliCreateLabelMultiple(c *check.C) {
 		c.Assert(strings.TrimSpace(out), check.Equals, v)
 	}
 }
+
+func (s *DockerSuite) TestVolumeCliRmForce(c *check.C) {
```

Comment:
Thanks @cpuguy83. I was wondering about how to write a test case for the scenario before. Let me update the pull request to add the suggested test.